### ISSUE TITLE
[RFE] Allow using env variables to set runtime configuration for `local-gadget`

### ISSUE THREAD

**Issue by blanquicet** [inspektor-gadget/inspektor-gadget#1044]
## Description

Currently, it is necessary to set the runtimes to be used and their socket paths at each `local-gadget` call:

```bash
$ local-gadget list-containers --containerd-socketpath=/bar/foo --runtimes=containerd
$ local-gadget snapshot process --containerd-socketpath=/bar/foo --runtimes=containerd
$ local-gadget trace exec --containerd-socketpath=/bar/foo --runtimes=containerd
```

## Impact

It is cumbersome for users to have to specify it every time.

## Ideal future situation

`local-gadget` provides the possibility of using env variables to set them:

```bash
$ export RUNTIMES=containerd
$ export CONTAINERD_SOCKETPATH=/bar/foo
$ local-gadget list-containers
$ local-gadget snapshot process
$ local-gadget trace exec
```

It seems it's possible by using viper + cobra https://github.com/kubereboot/kured/pull/464/files.

**Comment by pawarpranav83** [comment]
Currently working on this. Will create a PR soon!

**Comment by mauriciovasquezbernal** [comment]
@mqasimsarfraz do you think it was fixed by the configuration file introduced in #2831?

**Comment by mauriciovasquezbernal** [comment]
Internally confirmed that it's indeed fixed by #2831
